### ISSUE TITLE
Wrong result with pytket-qulacs

### ISSUE THREAD

**Issue by georgios-ts** [CQCL/pytket#35]
```python
from numpy import pi
import pytket as tket
from pytket.qulacs import tk_to_qulacs
from qulacs import QuantumState

qc = tket.Circuit(1)
qc.Rx(0.5, 0)

qc = tk_to_qulacs(qc)

state = QuantumState(1)
qc.update_quantum_state(state)

print(state.get_vector())
----
Output:
[0.70710678+0.j         0.+0.70710678j]
Expected Output:
[0.70710678+0.j         0.-0.70710678j]
```

From Qulacs documentation:
*Rotation gates RX, RY, and RZ operate as Pauli rotation exp(𝑖𝜃/2𝑃) based on corresponding Pauli operator 𝑃 and argument  𝜃*

Since tket uses exp(-𝑖aπ/2𝑃) representation, arguments of rotation gates should be multiplied by -1 inside `tk_to_qulacs`.

**Comment by cqc-alec** [comment]
Thank you for the report. This is indeed a bug in the `pytket-qulacs` backend. We will fix it in the next release, due around the beginning of February.

(In the meantime if you want to patch your 0.3.3 installation you should change `param` to `-param` at line 73 of `qulacs_convert.py`.)

**Comment by ss2165** [comment]
Fixed in pytket-qulacs 0.4.0
